helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -125,13 +125,21 @@ def _onchange_dominion_user_id(self):
     # CRUD
     # ---------------------------------------------------
 
+    def _creation_subtype(self):
+        return self.env.ref("helpdesk_mgmt.hlp_tck_created")
+
     @api.model
     def create(self, vals):
         if vals.get("number", "/") == "/":
             vals["number"] = self._prepare_ticket_number(vals)
         if not vals.get("team_id") and vals.get("category_id"):
             vals["team_id"] = self._prepare_team_id(vals)
-        return super().create(vals)
+        res = super().create(vals)
+        if res.team_id:
+            res.message_subscribe(
+                partner_ids=res.team_id.message_follower_ids.partner_id.mapped("id")
```

Comment:
I think this is safer

```suggestion
                partner_ids=res.team_id.message_follower_ids.mapped("partner_id.id")
```